onboarding_documentation/technical_documentation/analyst_tutorial/analyst_tutorial.md

```diff
@@ -0,0 +1,122 @@
+Our aim is to validate the pipeline's functionality and output by running it on a subset of the bioheart dataset. To do this we will use the script [create_test_subset.py](https://github.com/populationgenomics/metamist/blob/f6c226d08a8ee9875014d8c99cfe119742221efb/scripts/create_test_subset.py) in the metamist repository. This script selects a random set of samples from `bioheart` and copies their read files to the `bioheart-test` bucket. Execute the following command after navigating to the main branch of your metamist repository in the CLI.
+
+To run this script we must first be in the `metamist` repository. So navigate to your `metamist` repository, ensure you're on the `main` branch of `metamist` and then run the following command from CLI:
+
+**Note: We will need to set up more genomes in the actual bucket we're using**
+```bash
+analysis-runner \                     
+--dataset bioheart --description "populate bioheart test subset" --output-dir "bioheart-test" \
+--access-level full \
+scripts/create_test_subset.py --project bioheart --samples XPG280371 XPG280389 XPG280397 XPG280405 XPG280413 --skip-ped
+```
+**FOR REFERENCE: The above was taken from [this](https://centrepopgen.slack.com/archives/C03FA2M1MR9/p1700020527448029?thread_ts=1699935103.776929&cid=C03FA2M1MR9) Slack thread**
+
+
+## 2. Writing a config file
+
+Config files are used to specify the parameters of an analysis job. They are written in TOML format. There is already a great description of config files and how they're used in the `team-docs` repo [here](https://github.com/populationgenomics/team-docs/blob/13755bd51356b50ce11e6be78a76e53ed0a3ccb1/cpg_utils_config.md).
+
+In short; config files contain all the parameters needed to run the job we want such as images, references, specific stages of the pipeline to run, and the input data. They are required for the `analysis-runner` to start.
+
+When running stages in the `large cohort` pipeline there are default config files that are automatically chosen (see the default [here](https://github.com/populationgenomics/production-pipelines/blob/0bcf9775206f10ee91ac197c8c178f844ecad447/cpg_workflows/defaults.toml)) with subsequent user-defined parameters overriding these defaults as necessary. The default config file for the `large cohort` pipeline can be found [here](https://github.com/populationgenomics/production-pipelines/blob/main/configs/defaults/large_cohort.toml) and in it you can see a description of some of the parameters and what they're used for. 
+
+#### Task: Write a config file
+- If we are wanting to run the `large cohort` pipeline on the `bioheart-test` dataset, we will need to create a config file that is capable of doing this.
+- Have a go at writing your own config file capable of running the `large cohort` pipeline on `bioheart-test` up until the `Combiner` stage. 
+- You can use the default config file as a starting point, and then override the necessary parameters to run the pipeline on `bioheart-test`.
```

Comment:
> You can use the default config file as a starting point

I personally think the default is very confusing and missing lots of entries. Plus, most of it is not explained well. It would be very hard for the individual going through this pipeline to know what they need to keep and omit without any background